src/platforms/flutter/upload-debug.mdx

```diff
@@ -18,32 +18,53 @@ For Flutter Desktop (Windows/macOS/Linux) `split-debug-info` and `obfuscate` fla
 
 Errors raised from the native layer in Flutter apps require certain debug information files to be uploaded. For example, an Android app can use `proguard` for minification and obfuscation. And when using NDK, dwarf debug files need to be uploaded. Flutter Web requires sourcemaps and iOS apps also require dwarf debug information files.
 
-## Sentry Dart Plugin
+## Automatically Upload Debug Symbols
 
-[Try out the Sentry Dart Plugin](https://github.com/getsentry/sentry-dart-plugin), which uploads debug symbols automatically for you, or follow the manual steps below.
+The most straight forward way is to [use the Sentry Dart Plugin](https://github.com/getsentry/sentry-dart-plugin), which uploads debug symbols automatically for you.
 
-## Uploading for iOS and macOS
+### :clipboard: Install
```

Comment:
How does `:clipboard:` work? we don't use this anywhere else

Reply:
Ah, this is an emoji on gh. 🤦